tests/test_utilities.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+
+import unittest
+import os
+from unittest.mock import patch
+from settings import org, urls
+from src.utilities import get_repo_id, _get_repo_url, check_for_git_config
+
+
+def mocked_response(*args):
+    """Create class to mock requests response."""
+
+    class MockResponse:
+        def __init__(self, json_data, status_code, reason):
+            self.json_data = json_data
+            self.status_code = status_code
+            self.reason = reason
+
+        def json(self):
+            return self.json_data
+
+    # Careful, args needs to be a tuple, and that always ends with a "," character in Python!!
+    if args == ('https://api.github.com/repos/DataBiosphere/azul',):
```

Comment:
why is args a tuple if we are only ever passing one argument, either for 'azul' or 'foobar'?